src/app.ts

```diff
@@ -1,8 +1,9 @@
 import express from 'express';
 import pinoHttp from 'pino-http';
 import { rootRouter } from './routers';
-import { logger } from './logger';
+import { getLogger } from './logger';
 
+const logger = getLogger(__filename);
```

Comment:
The `getLogger` function and its use is also excellent and so much better than having to copy/paste and then remember to modify!
* `getLogger` is a more targeted import (one thing instead of two).
* The logic for which key to use is in one place (`logger.ts`) instead of spread throughout many files.
* The use of `__filename` is safely copied and pasted.
* The `__filename` tells us exactly what we want to know: "where did this message come from!?"
* The choice of whether to use a child logger or recycle another logger is also in one place (`logger.ts`) instead of spread throughout many files.

I like this so much more.